hudi-common/src/main/java/org/apache/hudi/common/model/HoodieRecord.java

```diff
@@ -18,14 +18,21 @@
 
 package org.apache.hudi.common.model;
 
+import org.apache.avro.Schema;
```

Comment:
this class needs to be avro free ultimately?

Reply:
All HoodieRecord and its subclass use avro schema.

Reply:
Avro schema exists as the unified schema for Hoodie. Spark record will convert Avro schema into structtype (cached) and use it.